### ISSUE TITLE
Google engine almost always gets 429 (too many requests) response

### ISSUE THREAD

**Issue by sgelis** [searx/searx#2854]
**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**
Version `1.0.0` from the official `searx/searx` repo.

**How did you install Searx?**
I followed the official Wiki instructions and ran the install scripts to get a regular Filtron-Searx-Morty install.

**What happened?**
When running a query through Google, I almost always (let's say 9 times out of 10) get a 429 (too many requests) response from Google.

Inspecting logs from Searx (see screenshots section) shows that Searx indeed gets a 429 response from Google, redirecting to a URL in the form of `https://www.google.com/sorry/index?continue=foo`.

**How To Reproduce**
1. Setup a Searx instance as per the Wiki, following the Filtron-Searx-Motry install through util scripts.
2. Run a query with the Google engine activated, possibly forcing a Google-only request by using the `!go` shortcut.

**Expected behavior**
I expect to be able to get Google results properly when I'm not trying to DDoS it :-)

**Screenshots & Logs**
![searx_google_2_anonymized](https://user-images.githubusercontent.com/74539800/122797558-e1929a80-d2bf-11eb-95e8-788208f009fe.png)

**Additional context**
My public IP and the public IP of the server hosting my Searx instance are not (to the best of my knowledge) being used to query Google **at all** outside of Searx..

This bug occurred since the very first request I sent using Searx. I setup Searx a couple days ago.

**Comment by unixfox** [comment]
Duplicate of https://github.com/searx/searx/issues/729

Unfortunately, someone from your range IP address is querying google a lot and this is affecting your IP address too. Searx can't really do anything about it, you may try to change hosting provider in order to have a new IP address that could potentially not be blocked by Google.

**Comment by crabvk** [comment]
This happen to me when I set
```ini
search:
    autocomplete : "google"
```
in the config.
